war/src/main/java/org/exoplatform/addons/es/EmbeddedESStartupServlet.java

```diff
@@ -27,11 +28,20 @@
 
   private static final Log LOG = ExoLogger.getLogger(EmbeddedESStartupServlet.class);
 
+  public static final String ES_EMBEDDED_ENABLED_PROPERTY_NAME = "exo.es.embedded.enabled";
```

Comment:
We could. I prefer to expose this variable so that others can use it, instead of seeing others classes duplicating it.